### PR TITLE
Remove <Nullable>disable/annotations</Nullable> from test projects

### PR DESCRIPTION
**Pull request by stephentoub**
Most test projects don't have them.  Removing a few stragglers.

### CODE REVIEW

File: src/libraries/System.ComponentModel.Annotations/tests/System.ComponentModel.Annotations.Tests.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFrameworks>$(NetCoreAppCurrent);net48</TargetFrameworks>
-    <Nullable>disable</Nullable>
+    <Nullable>disable</Nullable> <!-- Disable nullable attributes as some tests depend on them not being present. -->
```
Comment (eerhardt) [inline src/libraries/System.ComponentModel.Annotations/tests/System.ComponentModel.Annotations.Tests.csproj:5]: (nit) is VS going to mess with this comment placement?

**Review by eerhardt — APPROVED** [review]
LGTM. Thanks.

File: src/libraries/System.ComponentModel.Annotations/tests/System.ComponentModel.Annotations.Tests.csproj (hunk 1 above)
Comment (stephentoub) [reply src/libraries/System.ComponentModel.Annotations/tests/System.ComponentModel.Annotations.Tests.csproj:5]: Probably. :(

Comment (eerhardt) [reply src/libraries/System.ComponentModel.Annotations/tests/System.ComponentModel.Annotations.Tests.csproj:5]: Agreed. I wish VS didn't do it... especially when writing Tasks or lining up XML attributes. I'm sure that feedback is somewhere, I should probably go upvote it.

**Pull request merged**